src/components/content/order/formDataHelpers/getNamespaceByVersionAndCspAndServiceHostTypeHelper.ts

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
rename the file as 'serviceNamespaceHelper.ts'

Reply:
fixed.